tests/unit/ReportUtilsTest.ts

```diff
@@ -975,6 +975,22 @@ describe('ReportUtils', () => {
         });
     });
 
+    describe('getQuickActionDetails', () => {
```

Comment:
Thanks for adding this test!

Can you please follow [this format](https://github.com/Expensify/App/blob/4833a6c1cb39db9034843230ffab5455a112f1e2/tests/README.md#documenting-tests) to add a little bit of context and documentation to the test?

Reply:
@tgolen i updated

Reply:
OK, close enough, but that format is meant to have each comment start with literal "Given", "When" and "Then" words :D

Reply:
Noted for the feature.